package.json

```diff
@@ -17,10 +17,13 @@
     "test": "react-scripts test"
   },
   "dependencies": {
+    "@carbon/icons-react": "^10.5.0",
+    "carbon-components-react": "^7.5.1",
```

Comment:
You're missing the carbon-components import. see https://www.carbondesignsystem.com/tutorial/react/step-1/#install-carbon

Reply:
Yea, I'm going to have to start again because I had already moved on to the second part and am now in a git muddle cause I tried switching branches and it didn't work.
Thanks